components/brave_news/browser/test/BUILD.gn

```diff
@@ -16,6 +16,7 @@ source_set("brave_news_unit_tests") {
     "//brave/components/brave_news/browser/direct_feed_controller_unittest.cc",
     "//brave/components/brave_news/browser/direct_feed_fetcher_unittest.cc",
     "//brave/components/brave_news/browser/feed_building_unittest.cc",
+    "//brave/components/brave_news/browser/feed_generation_info_unittest.cc",
```

Comment:
For some reason I thought we were meant to have separate test targets - I have a vague recollection of a Slack thread about it (but my memory is hazy).

Happy to change it because this has always seemed a bit weird to me but maybe I'll get @petemill & @goodov to weigh in.